XDA_Scraper_Selenium.py

```diff
@@ -11,35 +11,36 @@
 url = "https://www.xda-developers.com/"
 
 total = []
-for page in range(1,pages):
+for page in range(1, pages):
 
     driver.get(url)
-    
+
     sections = driver.find_elements_by_class_name("layout_post_2")
 
     for section in sections:
 
         # print(section.text)
         articles = section.find_elements_by_class_name('item_content')
-        
-        for article in articles:
 
+        for article in articles:
 
-            headline = str(article.find_element_by_tag_name('h4').text).replace(',', '')
+            headline = str(article.find_element_by_tag_name(
+                'h4').text).replace(',', '')
 
-            excerpt = str(article.find_element_by_class_name('the-excerpt').text).replace(',', '')
+            excerpt = str(article.find_element_by_class_name(
+                'the-excerpt').text).replace(',', '')
 
             author = article.find_element_by_class_name('meta_author').text
 
             date_posted = article.find_element_by_class_name('meta_date').text
 
-
-            new = ((headline,excerpt,author,date_posted))
+            new = ((headline, excerpt, author, date_posted))
             total.append(new)
 
         # print(driver.find_element_by_class_name('next').text)
         url = f'https://www.xda-developers.com/page/{pagepylint+1}/'
 
 driver.close()
-df = pd.DataFrame(total,columns=['Headline','Excerpt','Author','Date_Posted'])
+df = pd.DataFrame(total, columns=['Headline',
+                  'Excerpt', 'Author', 'Date_Posted'])
 df.to_csv('XDA_selenium_scrape.csv')
```

Comment:
you could also make this file name into a constant at the top  
OUT_FILE = ''XDA_selenium_scrape.csv' 
.... 
`df.to_csv(OUT_FILE)`

The benefit of this is it makes it easier to turn into a CLI later on.